src/components/VmModals/VmYesNoModal.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react'
```

Comment:
Please consider in context of the new popover confirmation introduced within https://github.com/oVirt/ovirt-web-ui/pull/127. I think, we can avoid this modal YesNo dialog.
Please note, the component is not yet finished (CSS polishing in progress ...)